packages/fx-core/src/plugins/resource/function/utils/depsChecker/common.ts

```diff
@@ -42,6 +42,7 @@ export const nodeNotSupportedForSPFxHelpLink = `${defaultHelpLink}#current-insta
 export const dotnetExplanationHelpLink = `${defaultHelpLink}#why-net-sdk-is-needed`;
 export const dotnetFailToInstallHelpLink = `${defaultHelpLink}#failed-to-install-net-core-sdk-v31`;
 export const dotnetManualInstallHelpLink = `${defaultHelpLink}#linux-only-the-toolkit-cannot-find-net-5-or-net-core-31-on-your-machine-please-install-it-manually`;
+export const dotnetNotSupportTargetVersionHelpLink = `${defaultHelpLink}#dotnet-not-support-target-version`;
```

Comment:
I think toolkit extension will add #ErrorCode for help link, we don't need to add it in plugin?